DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Validators/NullValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentValidation;
+using FluentValidation.Results;
+
+namespace GitHubExtension.Security.WebApi.Library.Validators
+{
+    sealed class NullValidator<T> : AbstractValidator<T>
```

Comment:
I think there are two ways and our team should choose one.
1.	When someone creates entity he must immediately create validator for this if it needed.
2.	When someone creates entity that don’t need validator he must don’t forget to write line of code
container.Register<IValidate<Entity>, NullValidator<Entity>>();

Now I understand that 2nd approach is better, cause we can find runtime error with tests. When in 1st approach we can validate not-valid entity if someone will forget to create validator.
Thanks.